### PR TITLE
[stable/traefik] Traefik accesslogs filters

### PR DESCRIPTION
**Pull request by kenichi-shibata**
#### What this PR does / why we need it:

It enables traefik accessLogs filters documented [here](https://docs.traefik.io/configuration/logs/)

#### Checklist

- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/traefik/templates/configmap.yaml [hunk 1]
```diff
@@ -147,6 +147,18 @@ data:
     filePath = "{{ .Values.accessLogs.filePath }}"
     {{- end}}
       format = "{{ .Values.accessLogs.format }}"
+    {{- if .Values.accessLogs.filters }}
+    [accessLog.filters]
+      {{- if .Values.accessLogs.filters.statusCodes }}
+      statusCodes = [{{ range $_, $value := .Values.accessLogs.filters.statusCodes }}"{{ $value }}",{{ end }}]
```
Comment (dtomcej) [inline stable/traefik/templates/configmap.yaml]:
```suggestion
      statusCodes = [{{ range $_, $value := .Values.accessLogs.filters.statusCodes }}{{ $value | quote }},{{ end }}]
```

**Review by dtomcej — CHANGES_REQUESTED** [review]

**Comment by rimusz** [comment]
chart needs a version bump

File: stable/traefik/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.52.3
+version: 1.52.4
```
Comment (mmatur) [inline stable/traefik/Chart.yaml]:
```suggestion
version: 1.52.5
```

**Review by mmatur — CHANGES_REQUESTED** [review]

**Pull request closed without merging**